src/portal/portal.ts

```diff
@@ -23,6 +24,17 @@ export class Portal {
     return data;
   }
 
+  async updateOrganization(
+    options: UpdateOrganizationOptions,
+  ): Promise<Organization> {
+    const { data } = await this.workos.put(
+      `/organizations/${options.organization}`,
```

Comment:
Can we destructure `organization` here as we don't need to be passing it up in the body?

```ts
const { organization: organizationId, ...payload } = options;

const { data } = await this.workos.put(
  `/organizations/${organizationId}`,
  payload
);
```